xdsl/dialects/builtin.py

```diff
@@ -579,11 +579,6 @@ class Float16Type(ParametrizedAttribute, MLIRType):
 f16 = Float16Type()
 
 
-@irdl_attr_definition
```

Comment:
This is actually something that is needed. I feel that the proper way to go about this not being used is adding a testcase.

Reply:
This is duplicate code of lines 140-142 :)

Reply:
Ah sure, then remove it :+1: